friskby/static/js/friskby.js

```diff
@@ -89,13 +89,15 @@ var FriskbyModuleExtensions = (function ( Friskby ) {
         var _timeSeriesGeoJSON = {};
         var _beginTime, _endTime;
 
+        var host = location.host;
+
         var apiUrl = {
-            sensor:             "http://friskby.herokuapp.com/sensor/api/sensor/",
-            sensorType:         "http://friskby.herokuapp.com/sensor/api/sensortype/",
-            sensorInfo:         "http://friskby.herokuapp.com/sensor/api/sensorinfo/",
-            device:             "http://friskby.herokuapp.com/sensor/api/device/",
-            location:           "http://friskby.herokuapp.com/sensor/api/location/",
-            measurementType:    "http://friskby.herokuapp.com/sensor/api/measurement_type/"
+            sensor:             "http://" + host + "/sensor/api/sensor/",
```

Comment:
Hmmm - in my head I was thinking we could replace this with a relative url:
```html
sensor : "/sensor/api/sensor/",
sensorType: "/sensor/api/snesortype/"
```

but that might have been just whishful thinking?